packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -14,28 +15,45 @@ import SuccessDialog          from '../Modals/success-dialog.jsx';
 import 'Sass/account-wizard.scss';
 import 'Sass/real-account-signup.scss';
 
-const ErrorModal = ({ message }) => (
-    <div className='account-wizard--error'>
-        <IconDuplicate />
-        <h1><Localize i18n_default_text='Whoops!' /></h1>
-        <p>
-            {localize(message)}
-        </p>
-        <a
-            href='https://www.deriv.com/help-centre/'
-            type='button'
-            className='btn btn--primary btn__medium'
-            target='_blank'
-            rel='noopener noreferrer'
-        >
-            <span className='btn__text'>
+const ErrorModal = ({ message, code, openPersonalDetails }) => {
+    return (
+        <div className='account-wizard--error'>
+            <Icon
+                icon={IconDuplicate}
```

Comment:
The `icon` prop takes in a string, the `IconDuplicate` should be registered inside `packages/core/src/Assets/icon.jsx`, and the import for `IconDuplicate` can be removed from here.